docs/content/faq.md

```diff
@@ -71,7 +71,7 @@ support partially uploading an object. You can't take an existing
 object, and change some bytes in the middle of it.
 
 It would be possible to make a sync system which stored binary diffs
-instead of whole objects like rclone does, but that would break the
+instead of whole objects like rsync does, but that would break the
```

Comment:
Hm.. I guess this depends on what you interpret the "like..." part to refer to, and probably the original was intended to be read with emphasis on "instead of the whole objects like rclone does". Maybe simply adding a comma would make that clearer?

> It would be possible to make a sync system which stored binary diffs`,`
> instead of whole objects like rclone does, but that would break the

Reply:
Maybe I'm nitpicking here, and please know that your attempt at making the documentation clearer is appreciated. :)

In my opinion your changed text have a similar ambiguity as the original:

Original version:

> It would be possible to make a sync system which stored binary diffs instead of whole objects like rclone does, ...

Your version:

> It would be possible to make a sync system which stored binary diffs instead of whole objects like rsync does, ...

I think adding a comma or restructuring the sentence a bit is necessary to avoid confusion, e.g. one of these:

1. > It would be possible to make a sync system which stored binary diffs, instead of whole objects like rclone does, ...

2. > It would be possible to make a sync system which stored binary diffs, instead of whole objects, like rsync does, ...

3. > It would be possible to make a sync system which stored binary diffs like rsync does, instead of whole objects, ...

(I like 1 and 3 better than 2)

What do you think?

Reply:
Clarity, as always. 

Ive been going over the code over the last few days and have a new appreciation of the project.

My (trivial) suggestion was only to aide understanding.

Three is excellent.

Reply:
Great. If you want to change to that and push, I can merge it.